uber/templates/registration/form.html

```diff
@@ -84,12 +84,18 @@
         }
     };
 
-    $(document).ready(function(){
```

Comment:
We want "Trusted" and "Assigned Depts" to only show up if the volunteering checkbox is checked.  That wasn't implemented before (it's how it worked last year, before the refactor, so this turns it back on).